ui-frontend/packages/catalog-ui-search/src/main/webapp/js/model/MetacardProperties.js

```diff
@@ -42,7 +42,15 @@ module.exports = Backbone.AssociatedModel.extend({
     return
   },
   getPoints(attribute) {
-    return this.getGeometries(attribute).reduce(
+    const geometryAttributeValues = this.getGeometries(attribute)
+    const geometries = []
+    geometryAttributeValues.forEach(
+      geometry =>
+        Array.isArray(geometry)
+          ? geometry.forEach(geo => geometries.push(geo))
+          : geometries.push(geometry)
```

Comment:
✏️ The logic here is saying if `geometry` is an array, then push `geometry`'s elements to `geometries` rather than itself. Is that correct?

If so, I think you could use `flatMap()` here if you wanted, something like

`geometry.flatMap(geo => geometries.push(geo))`

the `flatMap` will do nothing different to the non-array elements, but to the arrays, it will flatten their depth by 1